### PR TITLE
[13.0][MIG] web_listview_range_select: Migration to 13.0

### PR DESCRIPTION
**Pull request by ernestotejeda**
cc @Tecnativa TT26087

### CODE REVIEW

File: web_listview_range_select/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# © 2017 Onestein (<http://www.onestein.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "List Range Selection",
+    "summary": """
+        Enables selecting a range of records using the shift key
+    """,
+    "version": "12.0.1.0.0",
```
Comment (emagdalenaC2i) [inline web_listview_range_select/__manifest__.py]:
```suggestion
    "version": "13.0.1.0.0",
```

**Comment by emagdalenaC2i** [comment]
@ernestotejeda I hace created #1634 But I can close it

**Comment by pedrobaeza** [comment]
@emagdalenaC2i yours seems to not be working. Have you tried this one?

**Comment by pedrobaeza** [comment]
Tried on runbot and working, but needs the version switch in the manifest.

**Comment by ernestotejeda** [comment]
@emagdalenaC2i , @pedrobaeza , changes done.

**Review by pedrobaeza — APPROVED** [review]

**Review by emagdalenaC2i — APPROVED** [review]
LGTM

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**